packages/graphql/src/translate/queryAST/ast/operations/composite/CompositeAggregationPartial.ts

```diff
@@ -50,16 +53,28 @@ export class CompositeAggregationPartial extends QueryASTNode {
     }
 
     public getChildren(): QueryASTNode[] {
-        return [];
+        return filterTruthy([...this.authFilters]);
+    }
+
+    public addAuthFilters(...filter: AuthorizationFilters[]) {
+        this.authFilters.push(...filter);
+    }
+
+    protected getAuthFilterPredicate(context: QueryASTContext): Cypher.Predicate[] {
+        return filterTruthy(this.authFilters.map((f) => f.getPredicate(context)));
     }
 
     public getSubqueries(context: QueryASTContext): Cypher.Clause[] {
-        if (!context.target) throw new Error("No parent node found!");
+        if (!context.target) {
+            throw new Error("No parent node found!");
+        }
 
         let pattern: Cypher.Pattern;
         const targetNode = createNodeFromEntity(this.target, context.neo4jGraphQLContext);
         let target: Cypher.Node | Cypher.Relationship = targetNode;
 
+        const authFilters = this.getAuthFilterPredicate(context);
```

Comment:
To avoid ambiguity with `this.authFilters`
```suggestion
        const authFilterPredicates = this.getAuthFilterPredicate(context);
```